srcpkgs/usql/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'usql'
 pkgname=usql
-version=0.9.5
+version=0.10.0
 revision=1
 build_style=go
 go_import_path=github.com/xo/usql
```

Comment:
usql --version doesn't report the correct version
```suggestion
go_import_path=github.com/xo/usql
go_ldflags="-X github.com/xo/usql/text.CommandVersion=${version}"
```

Reply:
Thanks I have updated the template